ZuluContent/Mobiles/Townfolk/Townfolk/AnimalTrainer.cs

```diff
@@ -95,7 +95,7 @@ public void EndStable(Mobile from, BaseCreature pet)
             {
                 SayTo(from, 1042562); // You do not own that pet!
             }
-            else if (pet.Summoned)
+            else if (pet.Summoned || pet.AI == AIType.AI_Familiar)
```

Comment:
is this mainly because the familiar should drop loot when it's killed? Otherwise I'd say just go with Summoned on the fam

Reply:
Yeah `BaseCreature.Summoned` has behaviour associated with it which doesn't fit this use-case, e.g. Mage AI trying to dispel, no loot dropping etc.

We can add a specific speech line but I don't think it'd add much value.

Reply:
ok